### PR TITLE
Skill ranks query added.

### PR DESCRIPTION
**Pull request by villanuevawill**
@c-o-l-o-r @codeluggage

### CODE REVIEW

File: sql_jobs/hourly/skill_ranks.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    id bigint,
+    normalized_name varchar(128),
+    name varchar(128),
+    total_count numeric
+);
+
+CREATE TABLE skill_ranks_tmp
+AS SELECT * FROM skill_ranks;
+
+DELETE FROM skill_ranks_tmp;
```
Comment (codeluggage) [inline sql_jobs/hourly/skill_ranks.sql]: Apparently truncate is slightly faster when deleting with no clause like this, but speed different is very likely insignificant.

File: sql_jobs/hourly/skill_ranks.sql [hunk 2]
```diff
@@ -0,0 +1,72 @@
+CREATE TABLE IF NOT EXISTS skill_ranks (
+    id bigint,
+    normalized_name varchar(128),
+    name varchar(128),
+    total_count numeric
+);
+
+CREATE TABLE skill_ranks_tmp
+AS SELECT * FROM skill_ranks;
```
Comment (codeluggage) [inline sql_jobs/hourly/skill_ranks.sql]: If this grows large in the future, there might be a speed improvement around limiting this select, as we're throwing out everything but the schema anyway.

File: sql_jobs/hourly/skill_ranks.sql (hunk 1 above)
Comment (villanuevawill) [reply sql_jobs/hourly/skill_ranks.sql]: super interesting :). I definitely look forward to when we need to thinka bout speed more. :)

File: sql_jobs/hourly/skill_ranks.sql (hunk 2 above)
Comment (villanuevawill) [reply sql_jobs/hourly/skill_ranks.sql]: yeah -def makes sense.

**Pull request merged**